generators/app/templates/README.md

```diff
@@ -0,0 +1,54 @@
+# <%= titlecaseProjectName %>
+
+
+[![NPM version](http://img.shields.io/npm/v/<%= projectName %>.svg)](https://www.npmjs.org/package/<%= projectName %>)
+[![Build Status](https://travis-ci.org/cerner/<%= projectName %>.svg?branch=master)](https://travis-ci.org/cerner/<%= projectName %>)
+[![Stories in Ready](https://badge.waffle.io/cerner/terra-ui.com.svg?label=ready&title=Ready)](http://waffle.io/cerner/terra-ui.com)
+
+{insert description}
+
+- [Getting Started](#getting-started)
+- [Documentation](#documentation)
+- [Dependencies](#dependencies)
```

Comment:
I removed that from the readme. I don't think it adds any value.

https://github.com/cerner/generator-terra-module/pull/2/commits/7a433b2b9049c111ed330bf32209f205a82a7937